docs/source/API/core/policies/TeamHandleConcept.md

```diff
@@ -59,6 +63,12 @@ class TeamHandleConcept {
 
 ## Public Class Members
 
+### Nested aliases
+
+  * `execution_space`: specifies the [execution space](https://kokkos.github.io/kokkos-core-wiki/API/core/execution_spaces.html) associated to the team
```

Comment:
Why is it a full URL rather than some sort of relative path?

Reply:
Because I will have to convert the page to rst anyway, so I did not want to put a relative path that works for me only and then change it again. I will make it relative in rst directly

Reply:
I am a bit surprised that it is not possible regardless of whether we use rst or md but fine.

Reply:
it is possible, just that it would have two different syntaxes